internal_packages/phishing-detection/package.json

```diff
@@ -4,6 +4,11 @@
   "main": "./lib/main",
   "license": "GPL-3.0",
 
+  "appId": {
+    "staging": "829s742c6v8G4jqV6KHUzaWd",
+    "production": "K3Rxa487mjMP36zh7w7xvo34"
```

Comment:
Generated these IDs randomly which seems to be wrong. The sync engine lets me know that it doesn't like my ID. How do I create a valid one?